Packages/com.unity.inputsystem/Documentation~/ActionAssets.md

```diff
@@ -34,6 +28,11 @@ The Actions Editor which opens is identical to the [Actions Editor in the Projec
 
 ## Using Input Action Assets
 
+
+## Type-safe C# API Generation
+
+Input Action Assets have a feature that is not present for project-wide actions, which is that you can **generate a C# class** from your action definitions, which allow you to refer to your actions in a type-safe manner from code. This means you can avoid looking up your actions by string.
```

Comment:
I just generated it, so its definitely possible via asset Inspect menu. Might this text be from before we switched back to asset-based?

Reply:
Hence this should be described for Input Actions Asset in general, there is no different between assets since the option is only available from asset Inspector.

Reply:
Yes this is left-over from when you couldn't do this with project-wide assets

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)